rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -136,6 +136,16 @@ public static boolean isReleaseTx(BtcTransaction tx, Federation federation, Brid
         return false;
     }
 
+    public static boolean isMigrationTx(BtcTransaction btcTx, Federation activeFederation, Federation retiringFederation, Context btcContext, BridgeConstants bridgeConstants) {
+        if (retiringFederation == null) {
```

Comment:
This is a false assumption. Do we have it reported at least?

Reply:
Yes it's reported